documentation/testing.md

```diff
@@ -189,7 +189,7 @@ The three data-plane pods (`testpmd-app`, `trexconfig` and `job-trex-app`) execu
 In this baseline scenario, everything will be executed once the pipeline execution finishes. To check if the TRexApp job has passed or not, you can take a look at the `job-trex-app` logs and check the end:
 
 ```
-$ oc logs -n example-cnf job-trex-app-fcs5j
+$ oc logs -n example-cnf `oc get pod -n example-cnf|awk '{print $1}'|grep job-trex-app`
```

Comment:
This one is tricky, but I believe if the job exists this command should work if you want to try
```
oc -n example-cnf logs -l example-cnf-type=pkt-gen-app
```
or maybe:
```
$ oc -n example-cnf logs job/job-trex-app
```